### ISSUE TITLE
Two months display

### ISSUE THREAD

**Issue by flintforge** [freiksenet/react-calendar#40]
Hi and Thanks. Looks like a pragmatic component.
I tried to set size={2} but it still display a 12 month calendar. Is it possible to get a 2 month display ?

**Comment by lolJS** [comment]
Yeah probably something I overlooked. I'll see if I can get a patch out sometime next week.
